src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx

```diff
@@ -72,6 +72,11 @@ export const HomeEmergingPicksArtworksRail: React.FC<HomeEmergingPicksArtworksRa
                 signal_label: artwork.collectorSignals
                   ? getSignalLabel(artwork.collectorSignals)
                   : "",
+                signal_bid_count:
+                  artwork.collectorSignals?.auction?.bidCount ?? undefined,
```

Comment:
#minor / non-blocking nit / fyi -- `undefined` isn't something that's typically assigned, as undefined is undefined, by definition. Null however can be a value, and in this case is what you'd expect to see here and elsewhere if `bidCount` is falsy.

Reply:
Ah, in that case, no problem at all and don't worry about it 👍 This was more of a general coding idiom thing, but i'd lean on whatever is over in cohesion. Didn't think about that side.